test/suites/homeshick_dir.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load ../helper
+
+@test 'bash override homeshick_dir' {
+	castle 'dotfiles'
+	local result=$( HOMESHICK_DIR=$_TMPDIR/nowhere $HOMESHICK_FN 2>&1 >/dev/null )
+	echo "result=$result"
+	[[ "$result" =~ "/nowhere/" ]]
+}
+
+@test 'fish override homeshick_dir' {
+	[ $(type -t fish) = "file" ] || skip "fish not installed"
+	pwd
+	ls $TESTDIR
+	cmd="source ${HOMESHICK_FN_SRC%.sh}.fish; set HOMESHICK_DIR \"$_TMPDIR/nowhere\"; $HOMESHICK_FN"
```

Comment:
I don't quite understand how this works. Why would calling `$HOMESHICK_FN` output the homeshick path?